imap_processing/cdf/imap_cdf_manager.py

```diff
@@ -11,11 +11,28 @@
 
 
 class ImapCdfAttributes(CdfAttributeManager):
-    """Contains IMAP specific tools and settings for CDF management."""
+    """
+    Contains IMAP specific tools and settings for CDF management.
 
-    def __init__(self):
-        """Set the path to the config directory."""
-        super().__init__(Path(__file__).parent / "config")
+    Parameters
+    ----------
+    source_dir : Path
+            Source directory.
```

Comment:
```suggestion
    source_dir : Path
            Source directory.
```
```suggestion
    source_dir : Path
        Source directory.
```

Looks like this got double-indented